### DISCUSSION TITLE
Changelog not capturing all activity

### DISCUSSION THREAD

**Discussion by abolar** [netbox-community/netbox#9037] [General]
We have experienced multiple instances of change log either not being captured at all, or the details within the change log not accurate (missing a value change). This is not always the case. Every once in a while, when we update a device with a new serial_num (or any attribute for that matter), the device itself gets the updated. However, when we look at the change log, we sometimes see no change log at all, or sometimes see an entry in the change log, where the "before" and "after" looks exactly the same. Although the edit on the device itself goes through.
Our team heavily relies on the change logs.

Has anybody else encountered this before?

Netbox version: 3.0.11

**Comment by tmalkowski** [comment]
I've been running into the same issue, using netbox version 3.3.4.

In case anyone else runs across this like I did while searching, I believe this is related to the problem described here: https://github.com/netbox-community/netbox/issues/9439

tl;dr — edit gunicorn.py and set threads=1 (and increase workers accordingly)
